packages/openfeature-server-provider/package.json

```diff
@@ -8,12 +8,12 @@
     "@microsoft/api-extractor": "7.43.1",
     "@openfeature/core": "^1.1.0",
     "@openfeature/server-sdk": "^1.13.5",
-    "@spotify-confidence/sdk": "0.1.4",
+    "@spotify-confidence/sdk": "0.1.5",
     "rollup": "4.14.2"
   },
   "peerDependencies": {
     "@openfeature/server-sdk": "^1.13.5",
-    "@spotify-confidence/sdk": "^0.1.4"
+    "@spotify-confidence/sdk": "^0.1.5"
```

Comment:
In theory, the server provider should be able to stay on `0.1.4` but if I change that locally and run `bundle`, the constraints tool gets sad.